### ISSUE TITLE
Evaluation of fine-tuning models for metric depth estimation

### ISSUE THREAD

**Issue by li12138cmd** [DepthAnything/Depth-Anything-V2#177]
Thank you very much for your work. I noticed that in the metric depth process, I did not evaluate.py script as I did before V1. I implemented the evaluation of kitti verification set according to the code in trainer.
![image](https://github.com/user-attachments/assets/78545cca-a537-4683-90a2-de8f18837306)
This does not achieve the numerical accuracy in the paper, I used a fine-tuned model on vkitti, Images and labels provided for val.txt using validation data.model type vits.Can you provide more detailed code to explain this phenomenon?
![1728289393530](https://github.com/user-attachments/assets/c830211f-8500-45b0-ac78-708e8c0fcb14)

**Comment by Lixc23tsinghua** [comment]
Me too，
![微信图片_20241010205232](https://github.com/user-attachments/assets/20eaaf6b-2d8b-4800-b0e1-aee0b33b9fff)

**Comment by LiheYoung** [comment]
Hi, the metric depth models here are trained on Hypersim or Virtual KITTI 2. But the metric depth performance reported in the paper are from models trained on NYUv2 or KITTI, with the ZoeDepth model.

**Comment by MLgeek7325** [comment]
@LiheYoung hi, I roughly tested these three models(Depth-Anything-V2-Indoor(Hypersim)-Small/Base/Large), and their RMSE on NYUv2-valid were 0.693/0.753/0.744 respectively. Is this in line with expectations?

**Comment by vineyard0531** [comment]
I also use the vkitti data set for fine-tuning training. When I use the kitti data set for evaluation, the evaluation results get worse as the epoch increases. Have you ever encountered this situation?
![vkitti训练kitti测试评估结果](https://github.com/user-attachments/assets/014adf3a-221b-450e-9731-23986d4d51e7)
